src/vm/mod.rs

```diff
@@ -366,6 +366,28 @@ pub fn eval_all(
     })
 }
 
+#[derive(Debug, Clone, PartialEq, Copy, Serialize, Deserialize)]
+pub enum StacksNetworkType {
```

Comment:
Please remove -- we just use `true` or `false` for mainnet everywhere else in the codebase.  No need to introduce two different ways to represent the same concept.

Reply:
Yep-- I agree with this. We use a boolean `is_mainnet` everywhere else (which maybe we shouldn't have), so we should be consistent with that.